README.md

```diff
@@ -69,6 +69,55 @@ options for you and run the image using `podman`:
     ~/src/flathub/com.example.App/com.example.App.json
 ```
 
+### On Flathub
+
+Flathub runs this tool hourly for all Flatpak repos under [github.com/flathub](https://github.com/flathub). This means for those repos to update sources, simply ensure any `x-checker-data` you need is present [as described below](https://github.com/flathub/flatpak-external-data-checker#changes-to-flatpak-manifests). Note Flathub's hosted tool only checks the default branch.
+
+To stop Flathub's tool from checking your repo, add `"disable-external-data-checker": true` to `flathub.json` in the default branch.
+
+### Custom workflow
+
+Alternatively, you can use own workflow. This can be useful if e.g. wanting to update non-default branches.
+
+Put this yaml file under `.github/workflows`, e.g. put it in `.github/workflows/update.yaml`. Ensure to put the correct path to the manifest in the last line.
+
+<details><summary>GitHub update workflow</summary><br/>
+
+```yaml
+name: Check for updates
+on:
+  schedule: # for scheduling to work this file must be in the default branch
+  - cron: "0 * * * *" # run every hour
+  workflow_dispatch: # can be manually dispatched under GitHub's "Actions" tab 
+
+jobs:
+  flatpak-external-data-checker:
+    runs-on: ubuntu-latest
+
```

Comment:
originally this had:
```
container:
      image: ghcr.io/flathub/flatpak-external-data-checker
```
after `runs-on`.

But as far as I can tell, it isn't needed.

Reply:
Indeed, it's not needed unless you want to run `flatpak-external-data-checker` command directly inside the container.